### ISSUE TITLE
"Heating Coil Water Heating Electricity Energy" Output Unavailable Despite Model Having a Coil:WaterHeating:AirToWaterHeatPump:Pumped

### ISSUE THREAD

**Issue by chriswmackey** [NREL/EnergyPlus#9124]
Issue overview
--------------
[This section of the Input/Output reference](https://bigladdersoftware.com/epx/docs/9-5/input-output-reference/group-heating-and-cooling-coils.html#outputs-23), implies that we should be able to request the "Heating Coil Water Heating Electricity Energy" output whenever there's a `Coil:WaterHeating:AirToWaterHeatPump:Pumped` in the model.  However, this output does not seem to be available for some reason when the heating coil is a part of the heat pump water heater, which ultimately means that there's no way to get timeseries outputs of electricity usage by the heat pump.

### Details
I tried for a few hours to get the timeseries electricity usage of a HeatPumpWaterHeater out of EnergyPlus but I haven't been able to do so. Here is a simple 7-zone IDF file that has such a heater:
https://drive.google.com/file/d/1_ZHIbkKAYk07h2NRty6-HDhGnvCHjmwf/view?usp=sharing

The Input/Output reference implies that the "Heating Coil Water Heating Electricity Energy" output should be able to give us this but, as the .rdd file for the IDF file above shows, the output is not available. If it's truly not possible to request the "Heating Coil Water Heating Electricity Energy" output for this case, this begs the question of whether there any way to get the timeseries electricity usage of the heat pump for this case?

This issue is also reported on Unmethours:
https://unmethours.com/question/47805/tracking-hpwh-performance/

**Comment by shorowit** [comment]
The documentation appears to be incorrect; the output variables are "Cooling Coil Foo", not "Heating Coil Foo". We've been using "Cooling Coil Water Heating Electricity Energy" in our modeling workflow to successfully get the energy use.

Interestingly, note that the description of the Coil:WaterHeating objects all say:

> From the air-side heat transfer perspective, the water-heater receives heating energy from air. Thus it is considered as a cooling coil, and the output variable names are using “Cooling Coil”.

But then the output variables in the documentation don't reflect that statement.

So it looks like the output variables for [Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed](https://bigladdersoftware.com/epx/docs/9-5/input-output-reference/group-heating-and-cooling-coils.html#coil-waterheating-airtowaterheatpump-variablespeed) are correctly labeled as "Cooling Coil" while [Coil:WaterHeating:AirToWaterHeatPump:Pumped](https://bigladdersoftware.com/epx/docs/9-5/input-output-reference/group-heating-and-cooling-coils.html#coilwaterheatingairtowaterheatpumppumped) and [Coil:WaterHeating:AirToWaterHeatPump:Wrapped](https://bigladdersoftware.com/epx/docs/9-5/input-output-reference/group-heating-and-cooling-coils.html#coilwaterheatingairtowaterheatpumpwrapped) are incorrectly labeled.

I will add this issue to https://github.com/NREL/EnergyPlus/issues/9111.

**Comment by chriswmackey** [comment]
Thank you, @shorowit , for the quick response. I just verified that I can request "Cooling Coil Water Heating Electricity Energy" from the IDF that I linked to and it gives me the expected results of electric energy for the heat pump.

I can also understand why labeling it as a "Cooling Coil" might not have been the most intuitive name in the context of it being used to supply heat to a service hot water tank (I imagine this could be the origin of the error in the docs). Still, it's nevertheless a valid name for the output and updating the documentation should clear this up going forward.

Thanks again!
